core/src/test/java/kafka/server/share/DelayedShareFetchTest.java

```diff
@@ -704,6 +704,7 @@ public void testLocksReleasedForCompletedFetch() {
             .withShareFetchData(shareFetch)
             .withSharePartitions(sharePartitions1)
             .withReplicaManager(replicaManager)
+            .withPartitionMaxBytesStrategy(PartitionMaxBytesStrategy.type(PartitionMaxBytesStrategy.StrategyType.UNIFORM))
```

Comment:
Why the change only over here and not in other methods? Also should we mock the response from `PartitionMaxBytesStrategy`?